firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -326,7 +336,8 @@ public Task<HttpURLConnection> createRealtimeConnection() {
 
                 String installationAuthToken = installationAuthTokenTask.getResult().getToken();
                 setCommonRequestHeaders(httpURLConnection, installationAuthToken);
-                setRequestParams(httpURLConnection);
+                JSONObject requestBody = createRequestBody(installationIdTask.getResult());
```

Comment:
You could alternatively just pass `installationIdTask.getResult()` to `setRequestParams` and continue to call `createRequestBody` there.

Reply:
updated to pass both id and token to setRequestParams